api/config/cdr_config_local.json

```diff
@@ -107,7 +113,8 @@
       "creationTime": "2020-08-26 00:09:51Z",
       "numParticipants": 234525,
       "cdrDbName": "cdr",
-      "accessTier": "controlled"
+      "accessTier": "controlled",
+      "publicReleaseNumber": 4
```

Comment:
Some of the release numbers do not seem clear cut.

Reply:
Should be fine for Local/Test ... may require more thought elsewhere